PC/msvcrtmodule.c

```diff
@@ -664,10 +648,29 @@ PyInit_msvcrt(void)
                                                   _VC_CRT_BUILD_VERSION,
                                                   _VC_CRT_RBUILD_VERSION);
     st = PyModule_AddObject(m, "CRT_ASSEMBLY_VERSION", version);
-    if (st < 0) return NULL;
+    if (st < 0) return -1;
 #endif
     /* make compiler warning quiet if st is unused */
     (void)st;
 
-    return m;
+    return 0;
+
+}
+
+static PyModuleDef_Slot msvcrt_slots[] = {
+    {Py_mod_exec, exec_module},
+    {0, NULL}
+};
+
+static struct PyModuleDef msvcrtmodule = {
+    .m_base = PyModuleDef_HEAD_INIT,
+    .m_name = "msvcrt",
+    .m_methods = msvcrt_functions,
+    .m_slots = msvcrt_slots
```

Comment:
I prefer a trailing comma for all struct initialisers that could possibly be expanded in the future.

```suggestion
    .m_slots = msvcrt_slots,
```